### PR TITLE
Add links for Google AutoML operators

### PR DESCRIPTION
**Pull request by MaksYermak**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: tests/system/providers/google/cloud/automl/example_automl_dataset.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    )
+    # [END howto_operator_list_dataset]
+
+    # [START howto_operator_delete_dataset]
+    delete_dataset = AutoMLDeleteDatasetOperator(
+        task_id="delete_dataset",
+        dataset_id="{{ task_instance.xcom_pull('list_datasets_task', key='dataset_id_list') | list }}",
+        location=GCP_AUTOML_LOCATION,
+        project_id=GCP_PROJECT_ID,
+        trigger_rule=TriggerRule.ALL_DONE,
```
Comment (bhirsz) [inline tests/system/providers/google/cloud/automl/example_automl_dataset.py:166]: Nit: Keep trigger rule outside of the documentation markers (START / END)

File: tests/system/providers/google/cloud/automl/resources/__init__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```
Comment (bhirsz) [inline tests/system/providers/google/cloud/automl/resources/__init__.py:1]: Do we need ``__init__`` file in resources?

Comment (MaksYermak) [reply tests/system/providers/google/cloud/automl/resources/__init__.py:1]: @bhirsz this `__init__` file was added automatically by pre-commit script

File: tests/system/providers/google/cloud/automl/example_automl_dataset.py (hunk 1 above)
Comment (MaksYermak) [reply tests/system/providers/google/cloud/automl/example_automl_dataset.py]: @bhirsz I have deleted it

**Review by lwyszomi — APPROVED** [review]

**Pull request closed without merging**